portfolio/src/main/webapp/programming.html

```diff
@@ -0,0 +1,19 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>Programming</title>
+    <link rel="stylesheet" href="style.css">
+    <script src="script.js"></script>
+  </head>
+  <body>
+    <div>
+      <script src="nav.js"></script>
+    </div>
+    <p>In terms of programming, I love building everything, but I especially love games. 
+       One of my first big projects was a game of Chess, and some of my later projects have 
+       followed in a similar vein. I'm also really curious about Machine Learning, despite
+       not having any formal instruction in it, and I'm always ready to learn! For some of
+       my prouder projects, check out my <a href="http://github.com/K-Shao">Github</a>! </p>
```

Comment:
nit: this is pretty long, so it might be better to separate out the paragraph tags into separate lines